src/Common/tests/System/Net/PlatformDetection.Networking.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System
+{
+    public static partial class PlatformDetection
```

Comment:
`IsWindows10InsiderPreviewBuild16215OrGreater` and `IsUap` are in PlatformDetection.cs. 

Using partial class PlatformDetection.Networking.cs is better, since we can directly refer to their values.

Besides, I followed David's suggestion here: https://github.com/dotnet/corefx/pull/22189#discussion_r127114350